scopes/design/ui/alert-card/alert-card.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { ReactNode } from 'react';
+import { Card, CardProps } from '@teambit/base-ui.surfaces.card';
+import { LevelIcon, Level } from '@teambit/ui.elements.level-icon';
+import { backgrounds } from '@teambit/base-ui.surfaces.background';
+import { H6 } from '@teambit/documenter.ui.heading';
+import classNames from 'classnames';
+import { Separator } from '@teambit/ui.separator';
+import styles from './alert-card.module.scss';
+
+interface InfoCardProps extends CardProps {
+  /**
+   * title to be shown in the card
+   */
+  title: string;
+  /**
+   * Level of Icon, info, error, warning, success
+   */
+  level: Level;
+  /**
+   * children to be rendered within this component.
+   */
+  children: ReactNode;
+}
+
+export function AlertCard({ title, children, className, level, ...rest }: InfoCardProps) {
```

Comment:
why Alert card? plz rename to InfoCard

Reply:
It is an alert card because it gives an alert. the alert can be an info alert or a warning alert etc but it alerts you of something by giving you a level of alert by passing in the level prop